.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+## Issue that this pull request solves
+
+Closes: # (issue number)
+
+## Proposed changes
+
+Brief description of what is fixed or changed
+
+- Files changed
+- Dependencies if any (on other issues/PRs)
+- Basic tests done to validate
+- Conflicts if any (describe the reason for conflict)
+
+## Types of changes
+
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
```

Comment:
I don't know that contributors ever submit these types of changes. I would actually say we can remove "types of changes" section altogether - if they've read the contributor guidelines, they know to preface their commit with "type of change" such as `feat(content-server):` or `chore(docs)` which covers this.

Reply:
I will remove the section